ui/packages/catalog-ui-search/src/main/webapp/templates/geocoder.handlebars

```diff
@@ -12,8 +12,8 @@
  **/
  --}}
 <div class="cesium-viewer-toolbar">
-    <div class="geocoder-container">
+    {{!-- <div class="geocoder-container">
```

Comment:
Will we be able to delete this file now?

Reply:
This will be handled later when the map loading events get addressed.